secant/Dependencies/DeeplinkManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DeeplinkManager.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 15.06.2022.
+//
+
+import Foundation
+import _URLRouting
+import ComposableArchitecture
+
+struct DeeplinkManager {
+    enum Deeplink {
+        case home
+        case send(amount: Int64, address: String, memo: String)
+    }
+    
+    func resolveDeeplinkRequest(_ url: URL) throws -> Deeplink {
```

Comment:
I think this can be confused with a ZIP-321 ZEC Payment Request parsing section of code. 

```suggestion
    func resolveDeeplinkURL(_ url: URL) throws -> Deeplink {
```

Reply:
updated